modules/react/combobox/lib/ComboboxCard.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import {
+  createElemPropsHook,
+  createSubcomponent,
+  ExtractProps,
+} from '@workday/canvas-kit-react/common';
+import {Menu} from '@workday/canvas-kit-react/menu';
+
+import {useComboboxModel} from './useComboboxModel';
+
+export interface ComboboxCardProps<T = any> extends ExtractProps<typeof Menu.Card> {}
+
+export const useComboboxCard = createElemPropsHook(useComboboxModel)(model => {
+  return {
+    minWidth: model.state.width,
+  } as const;
```

Comment:
why is the `as const` needed here?

Reply:
It is not needed here for this case, but if the attribute was `position`, there would be a Typescript error saying something like "string is not assignable to `'relative' | 'absolute' | 'static' | ...`. The `as const` makes the object be readonly which narrows types. In this case, there's nothing to narrow.